ibis/pandas/execution/generic.py

```diff
@@ -875,8 +876,14 @@ def execute_node_where_scalar_scalar_series(op, cond, true, false, **kwargs):
 @execute_node.register(
     ibis.pandas.client.PandasTable, ibis.pandas.client.PandasClient
 )
-def execute_database_table_client(op, client, **kwargs):
-    return client.dictionary[op.name]
+def execute_database_table_client(op, client, timecontext, **kwargs):
+    df = client.dictionary[op.name]
+    if timecontext:
+        begin, end = timecontext
+        # filter with time context
+        mask = df[TIME_COL].between(begin, end)
```

Comment:
timecontext is inclusive of both endpoints right? is this in the docs

Reply:
Yes and I update doc to reflect this.